python/suricata/ctl/filestore.py

```diff
@@ -47,39 +50,39 @@ def register_args(parser):
         help="be quiet, log warnings and errors only")
     prune_parser.set_defaults(func=prune)
 
+
 def is_fileinfo(path):
     return path.endswith(".json")
 
+
 def parse_age(age):
-    m = re.match("(\d+)\s*(\w+)", age)
-    if not m:
+    matched_age = re.match(r"(\d+)\s*(\w+)", age)
+    if not matched_age:
         raise InvalidAgeFormatError(age)
-    val = int(m.group(1))
-    unit = m.group(2)
-
-    if unit == "s":
-        return val
-    elif unit == "m":
-        return val * 60
-    elif unit == "h":
-        return val * 60 * 60
-    elif unit == "d":
-        return val * 60 * 60 * 24
-    else:
+    val = int(matched_age.group(1))
+    unit = matched_age.group(2)
+    ts_units = ["s", "m", "h", "d"]
+    try:
+        idx = ts_units.index(unit)
+    except ValueError:
         raise InvalidAgeFormatError("bad unit: %s" % (unit))
+    multiplier = 60 ** idx if idx != 3 else 24 * 60 ** 2
```

Comment:
My only concern is this way of calculating the age. It feels more fragile to change than an if statement. but that being said I don't see a reason why we would need to extend this in the near future.